python_modules/libraries/dagster-dg/dagster_dg/cli/dev.py

```diff
@@ -132,7 +131,7 @@ def dev_command(
         cmd_location = "ephemeral dagster dev"
         temp_workspace_file_cm = _temp_workspace_file(dg_context)
     else:
-        exit_with_error("This command must be run inside a code location or deployment directory.")
+        exit_with_error("This command must be run inside a project or workspace directory.")
```

Comment:
dg-managed?

Reply:
I prefer "project or workspace directory" since the extent to which one of these is "dg-managed" is kinda at the discretion of the user.